docs/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -446,6 +446,13 @@
               </row>
 
               <row>
+                <entry>--keywords</entry>
+
+                <entry>Outputs a list of ECL reserved words to a file
```

Comment:
At present there is no way to change the name or location of this file. If I was to be pedantic "to a file (ECLKeywords.xml)" suggests that this is variable rather than if it said "to the file ECLKeywords.xml"